lib/topological_inventory/openshift/operations/processor.rb

```diff
@@ -87,16 +87,17 @@ def svc_instance_context_with_url(source_id, service_instance)
           }
 
           if provisioning_status(service_instance) == "ok"
-            url = svc_instance_url(source_id, service_instance)
+            svc_instance = svc_instance_by_source_ref(source_id, service_instance.spec&.externalID)
+            context[:service_instance][:id] = svc_instance.id unless svc_instance.nil?
```

Comment:
@eclarizio we can probably just `return if svc_instance.nil?` and save this other check https://github.com/ManageIQ/topological_inventory-openshift/pull/75/files#diff-fdaebc6a3309a594344f99864099165eR101

Reply:
Ah yeah good point. Then I think the `if url.present?` can likely go away as well, right?

Reply:
Yeah I think you're right, that cleans things up nicely